app/models/export/ProductDraftWrapper.java

```diff
@@ -0,0 +1,266 @@
+package models.export;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.products.*;
+import io.sphere.sdk.products.attributes.AttributeDraft;
+import io.sphere.sdk.producttypes.ProductType;
+import io.sphere.sdk.taxcategories.TaxCategory;
+import org.javamoney.moneta.Money;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.requireNonNull;
+
+public class ProductDraftWrapper extends Base {
+
+    private final LocalizedString name;
+    private final LocalizedString description;
+    private final LocalizedString slug;
+    private final ProductVariantDraftWrapper masterVariant;
+    private final List<ProductVariantDraftWrapper> variants;
+
+    public ProductDraftWrapper(@JsonProperty("name") final LocalizedString name,
+                               @JsonProperty("description") final LocalizedString description,
+                               @JsonProperty("slug") final LocalizedString slug,
+                               @JsonProperty("masterVariant") final ProductVariantDraftWrapper masterVariant,
+                               @JsonProperty("variants") final List<ProductVariantDraftWrapper> variants) {
+        this.name = requireNonNull(name);
+        this.description = requireNonNull(description);
+        this.slug = requireNonNull(slug);
+        this.masterVariant = requireNonNull(masterVariant);
+        this.variants = requireNonNull(variants);
+        this.variants.add(masterVariant);
+    }
+
+    public ProductDraft createProductDraft(final ProductType productType, final TaxCategory taxCategory) {
+        final Reference<ProductType> productTypeReference = productType.toReference();
```

Comment:
I suggest to use directly `Reference<ProductType>` and `Reference<TaxCategory>` as parameters.